### DISCUSSION TITLE
%E5%A5%BD%E6%96%87%E6%8E%A8%E8%8D%90/%E7%A8%8B%E5%BA%8F%E5%91%98%E8%84%B1%E5%8D%95%E6%8C%87%E5%8D%97

### DISCUSSION THREAD

**Discussion by giscus** [cpython666/cpython666.github.io#6] [Announcements]
# %E5%A5%BD%E6%96%87%E6%8E%A8%E8%8D%90/%E7%A8%8B%E5%BA%8F%E5%91%98%E8%84%B1%E5%8D%95%E6%8C%87%E5%8D%97

Python斗罗的代码笔记

https://cpython666.github.io/%E5%A5%BD%E6%96%87%E6%8E%A8%E8%8D%90/%E7%A8%8B%E5%BA%8F%E5%91%98%E8%84%B1%E5%8D%95%E6%8C%87%E5%8D%97.html

**Comment by kamechan001** [comment]
笑死，好一个脱单指南，祝你明年能脱单吧

**Comment by xtzg** [comment]
天才！

**Comment by CCCCOOH** [comment]
天才啊，你肯定单身吧

**Comment by Virginia-Zhang** [comment]
程序员可不都是男生啊切拜😂  这让程序媛们情何以堪😂

**Comment by dabaizhijia** [comment]
666
